HerPortal.UnitTests/HerPortal.UnitTests.csproj

```diff
@@ -7,17 +7,18 @@
     </PropertyGroup>
 
     <ItemGroup>
-        <PackageReference Include="FluentAssertions" Version="6.6.0" />
-        <PackageReference Include="Microsoft.EntityFrameworkCore.Relational" Version="6.0.16" />
-        <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.11.0" />
-        <PackageReference Include="Moq" Version="4.18.4" />
-        <PackageReference Include="NUnit" Version="3.13.2" />
-        <PackageReference Include="NUnit3TestAdapter" Version="4.0.0" />
-        <PackageReference Include="coverlet.collector" Version="3.1.0" />
+        <PackageReference Include="FluentAssertions" Version="6.6.0"/>
```

Comment:
This isn't a concern for us since the dependency is only used in the assertion library, within the unit tests, which won't be run on the server. The vulnerability itself is triggered by malicious payloads which in this instance would mean someone has access to the app server filesystem, in which case we'd have bigger concerns!

Not ideal but we are already working on omitting the unit test subproject which will sidestep the issue entirely.